### PR TITLE
Release notes and versioning

### PR DESCRIPTION
**Pull request by pgdr**
Update release notes

To be merged after #1250

### CODE REVIEW

**Comment by pgdr** [comment]
@joakim-hove comments that should be added to release notes?  If not, merge&release?

**Comment by joakim-hove** [comment]
I will update the release notes.

File: release-notes.txt [hunk 1]
```diff
@@ -1,3 +1,14 @@
+Versjon 2.0.12, October 2016
+
+* Made ecl_file_view 1st class citizen, exposed to API
+* Added EXCLUDE_HOSTS to LSF driver with dynamic blacklisting
+* Fixed MAX_RUNTIME is respected by WPRO
+* Added functionality to open restart files in RW mode with seek
+* Improved date handling (all is UTC now)
```
Comment (joakim-hove) [inline release-notes.txt]: As a result of this the filesystem version has been upgraded to 107, to use an old case you must upgrade it:  "ert_upgradefs_107  config_file".

**Comment by joakim-hove** [comment]
When the minor comment about filesystem upgrades has been adressed this should be merged. Thank you!

**Comment by pgdr** [comment]
@joakim-hove Thanks for the feedback, please see if I managed to do it correctly.

**Pull request merged**